content/content.js

```diff
@@ -92,7 +92,8 @@ function loadForemHTML(forems) {
     .forem-expanded {left: 0 !important;}
   `;
 
-  const homeLink = `<a href="https://www.forem.com/discover/" rel="noreferrer" title="Forem web site"><img src="https://res.cloudinary.com/practicaldev/image/fetch/s--eAO3JVgB--/c_limit%2Cf_auto%2Cfl_progressive%2Cq_auto%2Cw_880/https://www.forem.com/logo.png" style="${
+  const foremLogoUrl = chrome.runtime.getURL("content/forem-logo-512.png");
```

Comment:
This is how we get a local path to the bundled Forem logo

Reply:
This is a beautiful solution, to bundle the image along with the extension. Love it 👏